### ISSUE TITLE
LocalAutoSave and tinymce 4.3.12

### ISSUE THREAD

**Issue by stephanlenhart** [valtlfelipe/TinyMCE-LocalAutoSave#13]
Hi!

Today I upgraded my site to tinymce 4.3.12. With this version the LocalAutoSave doesn't seem to work anymore. Can you confirm?

BR
Stephan

**Comment by valtlfelipe** [comment]
Hi Stephan!

I just tested it on the latest version (4.3.12), and it worked as expected. I updated the demo to the newer version here http://valtlfelipe.github.io/TinyMCE-LocalAutoSave/, please have a look.

Could you please provide an example of the code that you having issues?

**Comment by stephanlenhart** [comment]
HI Felipe,

turned out that I used some coding that made it incompatible. Once I added following code to make tinymce also working together with a validation plugin:

`
setup: function(editor) {
  editor.on("change", function(e) {
      tinyMCE.triggerSave();
      $("#" + editor.id).valid();
   });
}`

It's the triggerSave() that made problems. But it seems it isn't needed at all by the validation. So I skip it for now.

Thanks for your immediate support!

Stephan
